src/infra/alarm/template.yaml

```diff
@@ -522,10 +522,10 @@ Resources:
       ActionsEnabled: true
       AlarmActions:
         - !Ref AlertingSNSHighAlertNotificationTopicARN
-      AlarmDescription: "DynamoDB query too many items error five times within a minute."
+      AlarmDescription: "DynamoDB query too many items error once within a minute."
       ComparisonOperator: GreaterThanOrEqualToThreshold
       DatapointsToAlarm: 1
```

Comment:
Seems `DatapointsToAlarm` can be removed. Not required if EvaluationPeriod is set to 1.